src/OpenGL/DummyTypes.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+namespace OpenToolkit.Graphics.GL
+{
```

Comment:
Why the need for Dummy types? Debugging purposes?

Reply:
No it's for the generated bindings. They rely on these types.